syntax = "proto3";

package mcp;

// ðŸ‘‡ LÃ­nea clave para que funcione la generaciÃ³n de cÃ³digo Go
option go_package = "mcp_eks_adapter/proto;mcp";

service ModelContextService {
  rpc GetContext (ContextRequest) returns (ContextResponse);
}

message ContextRequest {
  string cluster_name = 1;
}

message ContextResponse {
  string kubernetes_version = 1;
  repeated HelmChart charts = 2;
}

message HelmChart {
  string name = 1;
  string version = 2;
  string namespace = 3;
}
